core/issuers.js

```diff
@@ -120,9 +120,12 @@ Description must be truthy: ${description}`;
   // additional methods
   const issuer = makeCustomIssuer(coreIssuer);
 
-  const label = harden({ issuer, description });
-
+  // This module only suggests a label to makeAssay. But the actual
+  // label is what makeAssay says it is.
+  let label = harden({ issuer, description });
```

Comment:
Maybe if we called this "suggestedLabel" and then called what comes out of makeAssay "label" we might be able to get rid of the comment.